sklearn/mixture/gaussian_mixture.py

```diff
@@ -651,16 +645,15 @@ def _m_step(self, X, resp):
             self.covariances_, self.covariance_type)
 
     def _estimate_log_prob(self, X):
-        return {"full": _estimate_log_gaussian_prob_full,
-                "tied": _estimate_log_gaussian_prob_tied,
-                "diag": _estimate_log_gaussian_prob_diag,
-                "spherical": _estimate_log_gaussian_prob_spherical
-                }[self.covariance_type](X, self.means_,
-                                        self.precisions_cholesky_)
+        return _estimate_log_gaussian_prob(
+            X, self.means_, self.precisions_cholesky_, self.covariance_type)
 
     def _estimate_log_weights(self):
         return np.log(self.weights_)
 
+    def _compute_lower_bound(self, _, log_prob_norm):
+        return log_prob_norm
+
```

Comment:
In `_set_parameters` below, compute `self.covariances_` from `self.precisions_`.